packages/shared/src/components/widgets/PostUsersHighlights.tsx

```diff
@@ -219,6 +224,14 @@ export const UserHighlight = (props: UserHighlightProps): ReactElement => {
           source={user}
         />
       )}
+      {isUserType && (
+        <FollowButton
+          userId={id}
+          type={ContentPreferenceType.User}
+          status={(user as CommentAuthor).contentPreference?.status}
+          entityName={`@${handleOrUsernameOrId}`}
```

Comment:
Why entityName? I assumed this would be the `entity` name like `User` or `Source`?